### PR TITLE
Followup work on setup github actions

### PR DESCRIPTION
**Pull request by fisker**
Todo:

- [x] blocker https://github.com/prettier/prettier/pull/7432 & https://github.com/prettier/prettier/pull/7433
- [x] Coverage report
- [x] Standalone tests

#7321

### CODE REVIEW

**Comment by fisker** [comment]
Coverage report not added yet.

**Comment by lipis** [comment]
Let's start removing the common ones..

**Comment by fisker** [comment]
@lipis You mean azure pipelines?

**Comment by lipis** [comment]
Yes

**Comment by fisker** [comment]
How about remove it in a separate PR?
Do we need keep both for a while?

**Comment by lipis** [comment]
Yes maybe.. let's finalize the gh ones..

**Comment by fisker** [comment]
@lipis Please a `CODECOV_TOKEN`

**Comment by armano2** [comment]
@fisker as for now codecov-action is not going to work great due to bug with PR numbers
https://github.com/codecov/codecov-action/pull/53

**Comment by fisker** [comment]
@armano2 We havn't setup `CODECOV_TOKEN`, I think thats the reason it fails.

**Comment by armano2** [comment]
yes i know i'm just pointing out that PR id is currently not set correctly by codecov-actions, and it causes to get invalid/corrupted coverage reports as all pull requests are treated as they are merge to master.

**Comment by lipis** [comment]
What's the status with this one? Version 2.0 is almost ready.. would be nice to get this one in and get rid of Azure (or at least the dublicates)

**Comment by fisker** [comment]
> @lipis Please add a `CODECOV_TOKEN`

**Comment by fisker** [comment]
@armano2 Can you have a look at codecov-action script? The [docs says `CODECOV_TOKEN` is not required for public repos](https://github.com/codecov/codecov-action#usage). Am I doing something wrong?

**Comment by lipis** [comment]
Down to 2 failing ones..!

**Comment by fisker** [comment]
> Down to 2 failing ones..!

@lipis Sorry, I don't understand this.

**Comment by lipis** [comment]
I'm just saying that only two tasks are failing now.. :)

**Comment by fisker** [comment]
I don't know why coverage upload failed, it works on my fork https://github.com/fisker/prettier/pull/318/checks?check_run_id=512762223#step:6:1

And the post url seems the same

```diff
-https://codecov.io/upload/v4?package=bash-20200316-40b1c41&token=secret&branch=adjust-github-actions&commit=49353ab7ad70096772f068bf4bb1e0c553499b14&build=57188406&build_url=http%3A%2F%2Fgithub.com%2Ffisker%2Fprettier%2Factions%2Fruns%2F57188406&name=&tag=&slug=fisker%2Fprettier&service=github-actions&flags=&pr=318&job=
+https://codecov.io/upload/v4?package=bash-20200316-40b1c41&token=secret&branch=adjust-github-actions&commit=49353ab7ad70096772f068bf4bb1e0c553499b14&build=57188409&build_url=http%3A%2F%2Fgithub.com%2Fprettier%2Fprettier%2Factions%2Fruns%2F57188409&name=&tag=&slug=prettier%2Fprettier&service=github-actions&flags=&pr=7431&job=
```

**Comment by lipis** [comment]
All green! That means we are ready to get rid of Azure? or they are still doing something more than we do with GitHub?

**Comment by fisker** [comment]
@lipis All azure jobs covered, problem is github actions seems slower than azure, especially the one collect coverage.

**Comment by lipis** [comment]
I would still go with GitHub...

**Comment by fisker** [comment]
I use github  actions too.

Seems faster with `--maxWorkers`, 20mins vs 15mins, should be acceptable

**Comment by lipis** [comment]
Let me know when we can disable Azure :)

**Comment by lipis** [comment]
Just remove the .azure directory and we should be fine when you're ready.

**Comment by fisker** [comment]
Let's disable it in #7824 , shall we?

File: jest.config.js [hunk 1]
```diff
@@ -37,4 +37,5 @@ module.exports = {
     "jest-watch-typeahead/testname",
   ],
   reporters: ["default"].concat(ENABLE_TEST_RESULTS ? "jest-junit" : []),
+  verbose: true,
```
Comment (thorn0) [inline jest.config.js]: Could we remove this `verbose: true`? Now Jest prints too much when I run it locally.

Comment (fisker) [reply jest.config.js]: How about set to `isCI`, so `yarn test` will print less, and `yarn test --verbose` can still print more

Comment (fisker) [reply jest.config.js]: #7866

**Pull request merged**